website/package.json

```diff
@@ -1,42 +0,0 @@
-{
-  "name": "website",
-  "version": "0.4.0",
-  "private": true,
-  "type": "module",
-  "scripts": {
-    "dev": "NEXT_TELEMETRY_DISABLED=1 next dev",
-    "build": "NEXT_TELEMETRY_DISABLED=1 next build && next export -o hosting/files/",
-    "publish": "gh-pages -b website --dist .",
-    "start": "NEXT_TELEMETRY_DISABLED=1 next start",
-    "fix": "eslint . --ext js,md,mdx --fix",
-    "check:ts": "tsc --project tsconfig.json"
-  },
-  "dependencies": {
-    "@emotion/css": "^11.0.0",
-    "@emotion/server": "^11.0.0",
-    "clipboard": "^2.0.6",
-    "gray-matter": "4.0.2",
-    "next": "^12.1.4",
-    "polished": "^4.1.3",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "rehype-autolink-headings": "^6.1.1",
-    "rehype-slug": "^5.0.1",
-    "remark": "^14.0.2",
-    "remark-gfm": "^3.0.1",
-    "remark-html": "^15.0.1",
-    "remark-parse": "^10.0.1"
-  },
```

Comment:
Not sure if there'll be an impact, but we should probably rerun `yarn install` in case there are any dependencies that are unused